README.md

```diff
@@ -107,17 +107,19 @@ Close observers may discern that even these _builder_ components, conceptualized
+| [`c/productQuantitySelector`](force-app/main/default/lwc/productQuantitySelector)<br/>`├──`[`c/commonNumberInput`](force-app/main/default/lwc/commonNumberInput)<br/>`└──`[`c/productQuantitySelectorPopover`](force-app/main/default/lwc/productQuantitySelectorPopover)                                                                                                                                                                                                                                                                                                                                                | Displays a [`c/commonNumberInput`](force-app/main/default/lwc/commonNumberInput) field constrained by product quantity rules (PQR) if available/configured.                                                                                                            | PDP              | [:white_check_mark:](force-app/main/default/lwc/builderProductQuantitySelector) |
+| [`c/productVariantSelector`](force-app/main/default/lwc/productVariantSelector)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          | Displays options for selecting product variations.                                                                                                                                                                                                                     | PDP              | [:white_check_mark:](force-app/main/default/lwc/builderProductVariantSelector)  |
+| [`c/searchResults`](force-app/main/default/lwc/searchResults)<br/>`├──`[`c/commonButton`](force-app/main/default/lwc/commonButton)<br/>`├──`[`c/commonLink`](force-app/main/default/lwc/commonLink)<br/>`├──`[`c/productPricing`](force-app/main/default/lwc/productPricing)<br/>`├──`[`c/productQuantitySelector`](force-app/main/default/lwc/productQuantitySelector)<br/>`├──`[`c/searchProductCard`](force-app/main/default/lwc/searchProductCard)<br/>`├──`[`c/searchProductGrid`](force-app/main/default/lwc/searchProductGrid)<br/>`└──`[`c/searchPagingControl`](force-app/main/default/lwc/searchPagingControl) | Displays the search results                                                                                                                                                                                                                                            | Search/PLP       |      [:white_check_mark:](force-app/main/default/lwc/builderSearchResults)      |
```

Comment:
missing `c/commonModal`

Reply:
Yeah, that's a little blurry, because the modal is part of the builder component and the dependencies here are related to the presentation. I think we need another kind of listing for the future. But right now putting the modal into this tree/list would actually be wrong.